src/App.css

```diff
@@ -20,3 +20,13 @@
 .App-link {
   color: #09d3ac;
 }
+
+.shopping-list {
+  margin: auto;
+  width: 50vw;
+}
+
+.shopping-list ul {
+  display: inline-block;
+  text-align: left;
+}
```

Comment:
We shouldn't style the app yet but `<ul>` components were spanning the entire screen with bullet points at the edge. I think sprinkling in some small sane css could be nice.